src/modules/rppi_validate.hpp

```diff
@@ -60,6 +60,27 @@ inline RppLayoutParams get_layout_params(RpptLayout layout, Rpp32u channels)
             layoutParams.bufferMultiplier = 3;
         }
     }
+    else if(layout == RpptLayout::NCDHW)
```

Comment:
if code segment under (layout == RpptLayout::NCHW) is same as (layout == RpptLayout::NCDHW), please group them with an || instead of this